dictionaries/dict.json

```diff
@@ -4991,7 +4991,6 @@
 "AEFD/KAEUT": "advocate",
 "AEFD/KWRAER": "evidentiary",
 "AEFD/REU": "evidentiary",
-"AEFLD": "realized",
```

Comment:
The correct definition (`RAEFLD`) already exists in this dict, so just removing the misstroke.